plugins/periskop/src/components/PeriskopErrorsTable/PeriskopErrorsTable.tsx

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import moment from 'moment';
```

Comment:
we've actively put effort into moving away from moment for the entire repository actually

- https://backstage.io/docs/architecture-decisions/adrs-adr010
- https://backstage.io/docs/architecture-decisions/adrs-adr012

I hope you don't mind switching to luxon

Reply:
No problem!